src/cordova/test.action.mjs

```diff
@@ -0,0 +1,112 @@
+// Copyright 2022 The Outline Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import url from 'url';
+import os from 'os';
+import minimist from 'minimist';
+import path from 'path';
+import fs from 'fs/promises';
+import rmfr from 'rmfr';
+
+import {spawnStream} from '../build/spawn_stream.mjs';
+import {getRootDir} from '../build/get_root_dir.mjs';
+import {getSupportedOSTarget} from '../build/get_supported_os_target.mjs';
+
+const APPLE_ROOT = path.join(getRootDir(), 'src', 'cordova', 'apple');
+const APPLE_LIBRARY_NAME = 'OutlineAppleLib';
+
+async function getCordovaTestParameters(buildParameters) {
+  let {
+    _: [platform],
+    verbose,
+    osVersion,
```

Comment:
Do we need to specify all this parameters?

It's hard to tell what they are doing. It would be helpful to clarify what's for building and what's for the runner.